README.md

```diff
@@ -1,7 +1,10 @@
-rovercode User Interface
+[![Chat](https://img.shields.io/badge/chat-developer-brightgreen.svg?style=flat)](https://rovercode.zulip.com)
```

Comment:
I'm getting a `400` for this link. Is it supposed to be `rovercode.zulipchat.com`?

Reply:
Sorry, fixed!